### ISSUE TITLE
Beautiful soup with lxml gets ofx file ("The ofx file is empty!")

### ISSUE THREAD

**Issue by nathangrigg** [jseutter/ofxparse#76]
Tested on OS X 10.10 and Ubuntu 12.04.

I see that we switched to lxml in bd544ba, but even the test introduced in that merge fails for me. Looking deeper, beautiful soup is returning an empty document.

**Comment by hiromu2000** [comment]
I've got the same error on Ubuntu 14.04.

```
======================================================================
ERROR: testCDATATransactions (tests.test_parse.TestSuncorpBankStatement)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/test_parse.py", line 521, in testCDATATransactions
    ofx = OfxParser.parse(open_file('suncorp.ofx'))
  File "ofxparse/ofxparse.py", line 363, in parse
    raise OfxParserException('The ofx file is empty!')
OfxParserException: The ofx file is empty!
```

**Comment by jseutter** [comment]
I think this is fixed now.  If it still happens with the current ofxparse, can you reopen the issue?

**Comment by nathangrigg** [comment]
I did see this further, but I think it is fixed for good with #108
